index.html

```diff
@@ -132,5 +123,251 @@
+                            </svg>
+                        </div>
+                    </button>                  
+                </div>
+                <ul class="languages-list">
+                    <li>
+                        <img src="./icons/js.png" alt="">
+                        <h5>JavaScript</h5>                       
+                    </li>
+                    <li>
+                        <img src="./icons/html.png" alt="">
+                        <h5>HTML</h5>                       
+                    </li>
+                    <li>
+                        <img src="./icons/css.png" alt="">
+                        <h5>CSS</h5>                   
+                    </li>
+                </ul>
+            </li>
+            
+            <li class="skill-list-2">
+                <div class="skill-list-names">
+                    <h4>Frameworks</h4>
+                    <button>
+                        <div class="arrow-container">
+                            <svg width="8" height="15" viewBox="0 0 8 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd" d="M0.292893 0.439622C0.683417 0.0490974 1.31658 0.0490974 1.70711 0.439622L7.70711 6.43962C8.09763 6.83015 8.09763 7.46331 7.70711 7.85384L1.70711 13.8538C1.31658 14.2444 0.683417 14.2444 0.292893 13.8538C-0.0976311 13.4633 -0.0976311 12.8301 0.292893 12.4396L5.58579 7.14673L0.292893 1.85384C-0.0976311 1.46331 -0.0976311 0.830146 0.292893 0.439622Z" fill="#505F79" />
+                            </svg>
+                        </div>
+                    </button>
+                </div>    
+            </li>
+            <li class="skill-list-2">
+                <div class="skill-list-names">
+                    <h4>Skills</h4>
+                    <button>
+                        <div class="arrow-container">
+                            <svg width="8" height="15" viewBox="0 0 8 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd" d="M0.292893 0.439622C0.683417 0.0490974 1.31658 0.0490974 1.70711 0.439622L7.70711 6.43962C8.09763 6.83015 8.09763 7.46331 7.70711 7.85384L1.70711 13.8538C1.31658 14.2444 0.683417 14.2444 0.292893 13.8538C-0.0976311 13.4633 -0.0976311 12.8301 0.292893 12.4396L5.58579 7.14673L0.292893 1.85384C-0.0976311 1.46331 -0.0976311 0.830146 0.292893 0.439622Z" fill="#505F79" />
+                            </svg>
+                        </div>
+                    </button>
+                </div>    
+            </li>
+        </ul>
```

Comment:
- [ ] as been requested by the previous reviewer consider removing the background color totally  you gave those arrows  so as to make your design look like the one provided in the Figma design
- your design 👇 
![new](https://user-images.githubusercontent.com/60146030/221367778-d2b71cff-1fa9-49c3-97d2-f79169eb8054.PNG)
- Figma design 👇 
![ss](https://user-images.githubusercontent.com/60146030/221367990-1cc169e5-cb29-4408-93f5-151b7cd77e0c.PNG)